dir.props

```diff
@@ -128,9 +128,11 @@
     <TargetsWindows Condition="'$(BuildOS)' == 'Windows_NT'">true</TargetsWindows>
 
     <TargetsUnix Condition="'$(TargetsFreeBSD)' == 'true' or '$(TargetsLinux)' == 'true' or '$(TargetsNetBSD)' == 'true' or '$(TargetsOSX)' == 'true'">true</TargetsUnix>
+    <TargetsLinuxCross Condition="'$(TargetsLinux)' == 'true' and '$(CROSSCOMPILE)' != ''">true</TargetsLinuxCross>
```

Comment:
I dont think we need cross-compilation identification like this. We perform cross-compilation on Windows as well without requiring this.

See my comment above - if we can use a single function that defines the DistroRid correctly, then your change here is not required.